app/assets/stylesheets/modules/_sticky-element-container.scss

```diff
@@ -1,17 +1,6 @@
 .govuk-sticky-element {
-  @include transition (opacity, .3s, ease);
-  opacity: 1;
-
-  &--hidden {
-    opacity: 0;
-    pointer-events: none;
-
-    @include ie-lte(8) {
-      // XXX: Would be nice to `@include .visuallyhidden`.
-      position: absolute;
-      left: -9999em;
-    }
-  }
+  position: absolute;
+  bottom: 0;
 
   &--stuck-to-parent {
```

Comment:
Is this block used now? Can it be removed? I can't find any references to `stuck-to-parent`.